packages/dashboard-frontend/src/store/__mocks__/devWorkspaceBuilder.ts

```diff
@@ -10,33 +10,41 @@
  *   Red Hat, Inc. - initial API and implementation
  */
 
-import { IDevWorkspace } from '../../services/workspace-client/devworkspace/types';
+import devfileApi from '../../services/devfileApi';
 import getRandomString from '../../services/helpers/random';
 import { DevWorkspaceStatus, WorkspaceStatus } from '../../services/helpers/types';
 
 export class DevWorkspaceBuilder {
 
-  private workspace: IDevWorkspace = {
+  private workspace: devfileApi.DevWorkspace = {
     kind: 'DevWorkspace',
     apiVersion: 'workspace.devfile.io/v1alpha2',
     metadata: {
+      annotations: {},
+      labels: {},
       name: 'dev-wksp-' + getRandomString(4),
       namespace: '',
+      uid: 'uid-' + getRandomString(4)
     },
     spec: {
       started: false,
       routingClass: 'che',
       template: {},
     },
-    status: {
-      devworkspaceId: getRandomString(4),
-      mainUrl: '',
-      phase: 'STOPPED',
-    }
+  }
+
+  private uidToDevworkspaceId(uid: string): string {
```

Comment:
Why do we need these methods here if we are not going to use workspaces Id at all?

https://github.com/eclipse-che/che-dashboard/pull/329/files#diff-16bcd373a2ae55c977a1ebc3e2c967d2c90cca1acbb409e1b8e7feb214e14cbdR16

Reply:
The only purpose is to properly mock a devworkspace for the unit test.